### PR TITLE
Enable human in the loop

### PR DESCRIPTION
**Pull request by Tomperez98**

### CODE REVIEW

File: src/resonate/record.py [hunk 1]
```diff
@@ -54,7 +58,21 @@ def __init__(
...
+            self.retry_policy = None
+        elif isinstance(invocation.unit.fn, str):
+            self.retry_policy = invocation.opts.retry_policy or None
+        elif isgeneratorfunction(invocation.unit.fn):
+            self.retry_policy = invocation.opts.retry_policy or never()
+        else:
+            assert iscoroutinefunction(invocation.unit.fn) or isfunction(
+                invocation.unit.fn
+            )
+            self.retry_policy = invocation.opts.retry_policy or constant(
```
Comment (dfarr) [inline src/resonate/record.py:72]: what about exponential as the default?

Comment (dfarr) [inline src/resonate/record.py:72]: what about exponential as the default?

File: src/resonate/scheduler/scheduler.py [hunk 2]
```diff
@@ -496,10 +502,17 @@ def _process_rfc(self, record: Record[Any], rfc: RFC) -> None:
                 if self._blocked_only_on_remote(root.id):
                     self._complete_task(root.id)
 
-    def _get_data_from_rfi(self, rfi: RFI) -> dict[str, Any]:
+    def _get_data_tags_and_headers_from_rfi(
```
Comment (dfarr) [inline src/resonate/scheduler/scheduler.py:505]: Can we move this to the bottom? We could also name something simpler such as `_get_info_from_rfi`

File: src/resonate/scheduler/scheduler.py [hunk 3]
```diff
@@ -513,10 +526,11 @@ def _get_data_from_rfi(self, rfi: RFI) -> dict[str, Any]:
                 "args": rfi.unit.args,
                 "kwargs": rfi.unit.kwargs,
             }
+            tags = {"resonate:invoke": rfi.opts.send_to or "default"}
+            headers = None
         else:
             assert_never(rfi.unit)
-
-        return data
+        return (data, tags, headers)
```
Comment (dfarr) [inline src/resonate/scheduler/scheduler.py:533]:
```suggestion
        return (data, headers, tags)
```

I'm being super anal here but data and headers belong together so it'd be nice to have them in order

**Review by dfarr — CHANGES_REQUESTED** [review]

File: src/resonate/scheduler/scheduler.py (hunk 2 above)
Comment (Tomperez98) [reply src/resonate/scheduler/scheduler.py]: I didn't have any better name. 😆 

Fixed

File: src/resonate/record.py (hunk 1 above)
Comment (Tomperez98) [reply src/resonate/record.py:72]: I'll then set a maximum exponential backoff time in the option. Backoff up to 30secs, after that will be constant 30 secs?

How do you feel about it?

**Review by dfarr — APPROVED** [review]

**Pull request merged**